StRoot/StSpinPool/StJetEvent/StJetTrack.h

```diff
@@ -104,7 +108,10 @@ class StJetTrack : public StJetElement {
   double   mNSigmaTofProton;
   double   mNSigmaTofElectron;
 
-  ClassDef(StJetTrack,6); 
+  int      mIdTruth;
+  int      mQaTruth;
```

Comment:
```suggestion
  unsigned short mIdTruth;
  unsigned short mQaTruth;
```

https://www.star.bnl.gov/webdata/dox/html/classStMuTrack.html
StMuTrack::mIdTruth is UShort_t while StMuTrack::idTruth() is Int_t